.gitignore

```diff
@@ -8,11 +8,11 @@
 .bower.json
 .sizecache.json
 
-npm-debug.log
+npm-debug.log*
```

Comment:
Not entirely sure, working theory is it newer npm sometimes creates npm-debug files with timestamps at the end